Libraries/Opc.Ua.Client/CoreClientUtils.cs

```diff
@@ -234,10 +236,28 @@ int discoverTimeout
         /// Select the best supported endpoint from an
         /// EndpointDescriptionCollection, with or without security.
         /// </summary>
+        /// <param name="url">The discovery Url of the server.</param>
+        /// <param name="endpoints"></param>
+        /// <param name="useSecurity"></param>
+        [Obsolete("Use the SelectEndpoint with ApplicationConfiguration instead.")]
```

Comment:
I'm not familiar with the codebase - I'm just wondering about this ... why introduce and provide a new obsolete Method?

Reply:
It is present in the master branch